native_client/java/app/build.gradle

```diff
@@ -4,7 +4,7 @@ android {
     compileSdkVersion 27
 
     defaultConfig {
-        applicationId "org.mozilla.deepspeech"
+        applicationId "org.mozilla.voice.sttapp"
```

Comment:
nit: shouldnt this be `org.mozilla.voice.stt.app` if we follow the package?

Reply:
I don't think it makes sense for the example app to be a subpackage of the binding.